### DISCUSSION TITLE
Emscripten build ?

### DISCUSSION THREAD

**Discussion by nyue** [pboettch/json-schema-validator#323] [Q&A]
I'd like to try using json-schema-validator in a C++ application which will be compiled to WASM.

Does emscripten build works out of the box ?

If not, I might invest time to get it compiling.

**Comment by pboettch** [comment]
Emscripten, I don't even know what this is.

**Reply by nyue** [reply]
Its a compiler tool chain base on LLVM to compile C++ code to WASM (WebAssembly)

https://emscripten.org/index.html
